eclipse/eclipse-2020-03/gcp-eclipse-2020-03.target

```diff
@@ -1,37 +1,37 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <?pde?>
 <!-- generated with https://github.com/eclipse-cbi/targetplatform-dsl -->
-<target name="GCP for Eclipse 2020-03" sequenceNumber="1591024101">
+<target name="GCP for Eclipse 2020-03" sequenceNumber="1654794613">
   <locations>
     <location includeMode="planner" includeAllPlatforms="false" includeSource="true" includeConfigurePhase="false" type="InstallableUnit">
-      <unit id="org.eclipse.sdk.feature.group" version="4.14.0.v20191210-0610"/>
-      <unit id="org.eclipse.jdt.feature.group" version="3.18.200.v20191210-0610"/>
-      <unit id="org.eclipse.m2e.feature.feature.group" version="1.14.0.20191209-1925"/>
-      <unit id="org.eclipse.m2e.sdk.feature.feature.group" version="1.14.0.20191209-1925"/>
-      <unit id="org.eclipse.m2e.wtp.feature.feature.group" version="1.4.3.20191209-1849"/>
-      <unit id="org.eclipse.m2e.wtp.sdk.feature.feature.group" version="1.4.3.20191209-1849"/>
+      <unit id="org.eclipse.sdk.feature.group" version="4.15.0.v20200305-0155"/>
```

Comment:
These changes are odd: the minor version shouldn't have changed here.

Reply:
The [2019-12 target](http://github.com/GoogleCloudPlatform/google-cloud-eclipse/blob/master/eclipse/eclipse-2019-12/gcp-eclipse-2019-12.target) show the Eclipse SDK feature is 4.14.x, so 2020-03 should have been 4.15.x. It looks like there was an error when generating the original 2020-03 target.